### PR TITLE
Add ballot results for enterprise-beans 4.0 and activation 2.0

### PR DESCRIPTION
**Pull request by waynebeaton**
This PR adds ballot results for the plan reviews of Jakarta Enterprise Beans 4.0 and Jakarta Activation 2.0.

### CODE REVIEW

File: activation/2.0/_index.md [hunk 1]
```diff
@@ -32,20 +32,20 @@ summary: "Release for Jakarta EE 9"
...
+| Scott Stark, Mark Little                       | Red Hat             |   1  |
+| David Blevins, Cesar Hernandez                 | Tomitribe           |   1  |
+| Ivar Grimstad                                  | EE4J PMC            |   1  |
 | Alex Theedom                                   | Participant Members |      |
-| Werner Keil                                    | Committer Members   |      |
-|                                                | Total               |      |
+| Werner Keil                                    | Committer Members   |   1  |
+|                                                | Total               |   8  |
 
 The ballot was run in the [jakarta.ee-spec mailing list]()
```
Comment (kwsutter) [inline activation/2.0/_index.md]: @waynebeaton could we also include a link to the final ballot message in the public spec mailing list?

File: enterprise-beans/4.0/_index.md [hunk 2]
```diff
@@ -33,20 +33,20 @@ Jakarta Enterprise Beans defines an architecture for the development and deploym
...
+| Scott Stark, Mark Little                       | Red Hat             |   1  |
+| David Blevins, Cesar Hernandez                 | Tomitribe           |   1  |
+| Ivar Grimstad                                  | EE4J PMC            |   1  |
 | Alex Theedom                                   | Participant Members |      |
-| Werner Keil                                    | Committer Members   |      |
-|                                                | Total               |      |
+| Werner Keil                                    | Committer Members   |   1  |
+|                                                | Total               |   8  |
 
 The ballot was run in the [jakarta.ee-spec mailing list]()
```
Comment (kwsutter) [inline enterprise-beans/4.0/_index.md]: @waynebeaton could we also include a link to the final ballot message in the public spec mailing list?

File: activation/2.0/_index.md [hunk 3]
```diff
@@ -32,20 +32,20 @@ summary: "Release for Jakarta EE 9"
 
 [Jakarta Activation 2.0 Release Record and Plan]()
```
Comment (kwsutter) [inline activation/2.0/_index.md]: I know this wasn't part of your effort, but could you also include a link to the release record?  
https://projects.eclipse.org/projects/ee4j.jaf/releases/2.0/plan

Thank you!

File: enterprise-beans/4.0/_index.md [hunk 4]
```diff
@@ -33,20 +33,20 @@ Jakarta Enterprise Beans defines an architecture for the development and deploym
 
 [Jakarta Enterprise Bean 4.0 Release Record and Plan]()
```
Comment (kwsutter) [inline enterprise-beans/4.0/_index.md]: I know this wasn't part of your effort, but could you also include a link to the release record?
https://projects.eclipse.org/projects/ee4j.ejb/releases/4.0/plan

Thank you!

**Comment by waynebeaton** [comment]
Nits addressed.

**Pull request merged**